system/remote-management.nix

```diff
@@ -6,6 +6,8 @@
     # from the ext.dividat.com network.
     joinNetworks = [ "a09acf02330ccc60" ];
   };
+  # Prevent ZeroTier from running on startup
+  systemd.services.zerotierone.wantedBy = lib.mkForce [];
```

Comment:
> I am wondering whether it might be useful to make the zerotier data folder below non-persistent.

This seems good to me, done in https://github.com/dividat/playos/pull/65/commits/58bc8f92e74f83d50f41c55346a3c249cc6bdd57.

Demo showing a different addres at each reboot on an installed PlayOS:

https://user-images.githubusercontent.com/6768842/110952118-21072d80-8346-11eb-85cd-d6e4057de4ad.mp4

Reply:
Nice, works as expected. Tested on physical machine.